README.md

```diff
@@ -6,50 +6,54 @@
 
 A library for validating emails against several RFC.
 
-## Supported RFCs ##
+## Supported RFCs
 
 This library aims to support RFCs:
 
-* [5321](https://tools.ietf.org/html/rfc5321), 
-* [5322](https://tools.ietf.org/html/rfc5322), 
-* [6530](https://tools.ietf.org/html/rfc6530), 
-* [6531](https://tools.ietf.org/html/rfc6531), 
+* [5321](https://tools.ietf.org/html/rfc5321),
+* [5322](https://tools.ietf.org/html/rfc5322),
+* [6530](https://tools.ietf.org/html/rfc6530),
+* [6531](https://tools.ietf.org/html/rfc6531),
 * [6532](https://tools.ietf.org/html/rfc6532),
-* [1035](https://tools.ietf.org/html/rfc1035) 
+* [1035](https://tools.ietf.org/html/rfc1035)
 
 ## Supported versions
 
 **Current major version with full support is v3**
 
-| Version |  Released  |  EOL  | Only critical bug fixes | Full  |
-| :-----: | :--------: | :---: | :---------------------: | :---: |
-| v4.x    | 2023/01/07 |   -   |            X            |   X   |
-| v3.x    | 2020/12/29 |   -   |            X            |       |
-| v2.1.x  | 2016/05/16 |  YES  |                         |       |
-| v1.2    | 2013/19/05 |  YES  |                         |       |
+| Version |  Released  | EOL | Only critical bug fixes | Full |
+|:-------:|:----------:|:---:|:-----------------------:|:----:|
+|  v4.x   | 2023/01/07 |  -  |            X            |  X   |
+|  v3.x   | 2020/12/29 |  -  |            X            |      |
+| v2.1.x  | 2016/05/16 | YES |                         |      |
+|  v1.2   | 2013/19/05 | YES |                         |      |
 
+## Requirements
 
-## Requirements ##
+* PHP 8.1
+* [Composer](https://getcomposer.org) is required for installation
+* [Spoofchecking](/src/Validation/Extra/SpoofCheckValidation.php)
```

Comment:
Any reason for splitting this line in 3 on the readme?

Reply:
Yes, I'm aware of line length recommendations. And I do enforce them in code.
However, the readme and documentation (from my POV) are not tied to it as most browsers and/or editors will wrap the lines to fit the reader screen size/definition. Adding breaking lines would break that adaptation I believe.

So, yes, please undo them for the readme. More of a personal choice here. Thanks